src/index.js

```diff
@@ -842,6 +935,82 @@ export default function ({ types: t, template }) {
         catch (error) {
           buildCodeFrameError(path, error)
         }
+      },
+
+      VariableDeclaration(path, state) {
+        if (state.opts[SKIP_ASSERTS_OPTION]) {
+          return
+        }
+
+        const node = path.node
+        try {
+          node.declarations.forEach(declarator => {
+            const id = declarator.id
+
+            if (id.hasBeenTypeChecked) {
```

Comment:
not sure why you handle a `hasBeenTypeChecked` boolean, commenting it out doesn't affect the tests